content/post/2022-08-15-nonviolent-agile-retrospectives.md

```diff
@@ -0,0 +1,490 @@
+feelings: You will get a better and more honest outcome if everyone votes
+on topics they feel strongly about and not voting "tactically" where you
+assume others will vote too or even to support someone's agenda.
+
+Improve Collection and Discussion phase by expressing your feelings (with
+examples).
+
+When coming up with action items, check if you're willing to volunteer for
+carrying out the action. If you have uncomfortable feelings (e.g. if you
+feel hesitant, resistant, "disappointed", sad, angry or doubtful), this
+can be an indicator that the action item does not fulfill all your needs
+(see next section).
+
+
+### Needs
+
+Needs are the central element of Nonviolent Communication. The key
+distinction for needs is between **needs** and **strategies**. In
+Nonviolent Communication needs are *abstract* and sometimes even thought
+of as universal to every human being. Strategies are the things that
+fulfill needs. For most needs there is more than one strategy to fulfill
+it, although most people have a preferred strategy. For example you can
+fulfill your need for learning and growth through online courses, books,
+talking to expert and colleagues, etc. You can fulfill your need for an
+comfortable temperature through clothing, opening the windows, getting a
+fan, moving around to get warm or changing the AC settings. From the
+temperature example you can see that conflict arise over *strategies*
+(e.g. someone opening the window or turning the heating on), while people
+can agree on a common shared need (for a comfortable temperature).
+
+If you want to check if something is more of a need or more of a strategy,
+check if it involves a specific action, item, time, place or person. If
+that's the case, then it's probably not a need in the definition of
+Nonviolent Communication. There are many word-lists with needs on the
+internet (search for "NVC list of needs"). You can look for those lists to
+get an idea what other people consider abstract needs in the definition of
+NVC. What you shouldn't do is use those list as a prescriptive tool to
+judge other people's language, judging "this is nonviolent and this is
+not". Using the lists in this way will turn Nonviolent Communication
+principles from a helpful tool for getting connected to other people into
+a source of conflict.
+
+Similar to the "I feel ..." prefix, the "I need ..." prefix can also
+introduce a strategy. Especially the prefix "I need you to ..." is rarely
+expressing a need and is often a suggestion for a strategy.
+
+#### For facilitators
+
+When preparing for a retrospective, think of the different roles in a
+team: product owner, developer, UX person, architect, etc). Each role can
+have "recurring" needs that come from their tasks and interactions with
+other roles. Try to put yourself in the shoes of each role and come up
+with a "top 5" list of needs a person with that role might have. Talk to
+people in that role and check your assumptions.
```

Comment:
I find it important to note that these "recurring needs" are different from what one would find on a "NVC list of needs".

Not getting my patches reviewed might prevent me from getting my needs of "belonging" and "being seen" fulfilled, but most of all, it prevents me from getting my work done. In the role of a Dev, I have the recurring need to get my patches reviewed. But "belonging" and "being seen" are needs of me as a _human_, not of my role as a Dev.
As in: I would claim that everyone in every role has the need of belonging and being seen if they are part of a team.